catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/TextAttribute.java

```diff
@@ -85,7 +90,7 @@
       segment -> segment.getSecurityMetadata().getClassificationText()),
   TEXT_CLASSIFICATION_AUTHORITY_TYPE(
       "text-classification-authority-type",
-      "TSCA TP",
+      "TSCATP",
```

Comment:
It doesn't look like this should have a space:
http://www.gwg.nga.mil/ntb/baseline/docs/stdi0006/NCDRD_20Dec06.pdf.